# This message holds status of all AR600 robot power supplies:
#  - 48v
#  - 12v
#  - 8v (2 supplies)
#  - 6v (2 supplies)

TypeSupplyState S48
TypeSupplyState S12
TypeSupplyState S8_1
TypeSupplyState S8_2
TypeSupplyState S6_1
TypeSupplyState S6_2

